sources/lib/Resources/config/services/pomm.yml

```diff
@@ -12,11 +12,15 @@ services:
         class: 'PommProject\SymfonyBridge\Serializer\Normalizer\FlexibleEntityNormalizer'
         tags:
             - { name: serializer.normalizer }
-    pomm.property_info:
-        class: 'PommProject\SymfonyBridge\PropertyInfo\Extractor\PommExtractor'
+
+    pomm.property_type_info:
+        class: 'PommProject\SymfonyBridge\PropertyInfo\Extractor\TypeExtractor'
         arguments: ['@pomm']
         tags:
             - { name: "property_info.type_extractor" }
+    pomm.property_info:
+        alias: 'pomm.property_type_info'
+        deprecated: 'The "%service_id%" service is deprecated. Use "pomm.property_list_info" instead'
```

Comment:
Can you open an issue? If it’s a deprecated notice, it’s a good side effect but we may remove this in the future.